src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -40,23 +33,7 @@ class WebexMeetingWidget extends Component {
     } else if (!audioPermission || !videoPermission || !meeting.ID) {
```

Comment:
it's ok to send an empty meeting id to the meeting component, please remove `!meeting.ID` from this condition

Reply:
done